tests/tools/kopiaclient/kopiaclient.go

```diff
@@ -73,6 +75,43 @@ func (kc *KopiaClient) CreateOrConnectRepo(ctx context.Context, repoDir, bucketN
 	return errors.Wrap(err, "unable to open repository")
 }
 
+// ConnectRepo connects to an existing one if possible.
+func (kc *KopiaClient) ConnectRepo(ctx context.Context, repoDir, bucketName string) error {
+	st, err := kc.getStorage(ctx, repoDir, bucketName)
+	if err != nil {
+		return err
+	}
+
+	if iErr := repo.Connect(ctx, kc.configPath, st, kc.pw, &repo.ConnectOptions{}); iErr != nil {
+		return errors.Wrap(iErr, "error connecting to repository")
+	}
+
+	return errors.Wrap(err, "unable to open repository")
```

Comment:
Same suggestion here - wrap is unnecessary
```suggestion
	return nil
```

Reply:
done